x-pack/plugins/siem/server/lib/timeline/default_timeline_headers.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedTimeline } from '../../../common/types/timeline';
+
+export const defaultColumnHeaderType = 'not-filtered';
+
+export const defaultHeaders: SavedTimeline['columns'] = [
```

Comment:
I think it will be nice to add the columns parameter in our draft API, so users do not have to reset them all the time